build/debian-iptables/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+timeout: 10800s
+
+substitutions:
```

Comment:
how do we keep these values synchronized with the Makefile? should we at least add a comment in the Makefile pointing here?

Reply:
I think ideally, yes. But the problem of synchronization will always be there as long as the Makefile remains as the source of truth.